acceptance/tests/resource/ssh_authorized_key/query.rb

```diff
@@ -18,7 +18,7 @@
   on(agent, "cp #{auth_keys} /tmp/auth_keys", :acceptable_exit_codes => [0,1])
 
   step "(setup) create an authorized key in the #{auth_keys} file"
-  on(agent, "echo 'ssh-rsa mykey #{name}' >> #{auth_keys}")
+  on(agent, "echo '' >> #{auth_keys} && echo 'ssh-rsa mykey #{name}' >> #{auth_keys}")
```

Comment:
@MikaelSmith Oh I see - we're skipping this test, as it's blocked on a different issue... looking into if it'll pass or not.